index.js

```diff
@@ -46,6 +46,7 @@ Toolkit.run(async tools => {
       milestone: attributes.milestone
     })
 
+    core.setOutput('url', issue.data.html_url)
```

Comment:
While we're here, what do you think about adding `number` too? In case it needs to be used in some future API call, it'd be easier than parsing the URL.

Reply:
Yep, sounds good 👍